tests/Microsoft.Identity.Test.Unit.net45/CoreTests/CacheTests/MsalTokenCacheKeysTests.cs

```diff
@@ -51,6 +51,10 @@ public void MsalAccessTokenCacheKey()
             Assert.AreEqual("accesstoken-clientid-contoso.com-user.read user.write", key.iOSService);
             Assert.AreEqual("accesstoken-clientid-contoso.com", key.iOSGeneric);
             Assert.AreEqual((int)MsalCacheKeys.iOSCredentialAttrType.AccessToken, key.iOSType);
+
+            Assert.AreEqual(
+                key,
+                new MsalAccessTokenCacheKey("login.microsoftonline.com", "contoso.com", "uid.utid", "clientid", "user.read user.write"));
```

Comment:
can we have consts for these? or  i can go clean up things later :)

Reply:
+1 for constants since they're used throughout the file.